suite-native/forms/src/hooks/useField.ts

```diff
@@ -27,14 +28,18 @@ export const useField = ({ name, label, defaultValue }: UseFieldArgs) => {
         defaultValue,
     });
 
+    // Inspired by https://react-hook-form.com/advanced-usage#TransformandParse.
+    // Allows to parse/transform the value before it's set to the input.
+    const transformedValue = valueTransformer ? valueTransformer(value) : value;
```

Comment:
maybe we can have default value transformer that returns the input string and then below just use:

`value: valueTransformer(value)`

Reply:
less code, better readability, but up to your appetite to leave it or take it.

Reply:
fixed b702f9114e